libphidgets/CMakeLists.txt

```diff
@@ -1,11 +1,18 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libphidgets)
 
-find_package(catkin REQUIRED COMPONENTS rosbuild roslib mk rosbuild rospack)
+find_package(catkin REQUIRED COMPONENTS mk rospack)
```

Comment:
as `libusb-dev` is not used in the exported headers, I#ll also remove it from the `DEPENDS` tag.